core/arch/riscv64/riscv64.asm

```diff
@@ -322,8 +322,20 @@ GLOBAL_LABEL(back_from_native:)
 # if !defined(STANDALONE_UNIT_TEST) && !defined(STATIC_LIBRARY)
         DECLARE_FUNC(_start)
 GLOBAL_LABEL(_start:)
-/* FIXME i#3544: Not implemented */
-        ret
+        nop
+        mv      fp, x0  /* clear frame ptr for stack trace bottom */
+
+        CALLC3(GLOBAL_REF(relocate_dynamorio), 0, 0, sp)
+        mv      ARG2, x0
+        mv      ARG3, x0
+
+        /* Clear 2nd & 3rd args to distinguish from xfer_to_new_libdr */
+.L_start_invoke_C:
+        mv      fp, x0 /* clear frame ptr for stack trace bottom */
+        mv      ARG1, sp /* 1st arg to privload_early_inject */
+        jal     GLOBAL_REF(privload_early_inject)
+        /* shouldn't return */
```

Comment:
style: Prefer capitalized and punctuated full-line comments (the end-of-line ones here could also be cap+punct) in new code

Reply:
Done.